### PR TITLE
fix(staking): dashboard restyling

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

Title says

##### Ticket

YOMO-1868

### CODE REVIEW

File: apps/wallet-mobile/src/components/Icon/TotalReward.tsx [hunk 1]
```diff
@@ -4,32 +4,34 @@ import Svg, {Circle, G, Path} from 'react-native-svg'
...
           id="Combined-Shape"
-          fill="#F14D78"
```
Comment (michalszorad) [inline apps/wallet-mobile/src/components/Icon/TotalReward.tsx:32]: Both colors "#3154CB" and "#F14D78" have been replaced with the same variable name `color` will it not cause issues for the icons where they have been displayed?

Comment (banklesss) [reply apps/wallet-mobile/src/components/Icon/TotalReward.tsx:32]: these icons are only used on this screen 👍🏻

File: apps/wallet-mobile/src/legacy/Dashboard/StakePoolInfo.tsx [hunk 2]
```diff
@@ -94,30 +100,26 @@ export const useStakePoolInfoAndHistory = (
...
+      color: color.text_gray_medium,
       ...atoms.body_2_md_regular,
-      flex: 1,
-    },
-    bottomBlock: {
-      ...atoms.px_lg,
-      ...atoms.py_xl,
+      ...atoms.flex_1,
     },
+    bottomBlock: {},
```
Comment (stackchain) [inline apps/wallet-mobile/src/legacy/Dashboard/StakePoolInfo.tsx]: is it needed?
```suggestion
```

File: apps/wallet-mobile/src/legacy/Dashboard/StakePoolInfo.tsx [hunk 3]
```diff
@@ -94,30 +100,26 @@ export const useStakePoolInfoAndHistory = (
 const useStyles = () => {
   const {color, atoms} = useTheme()
   const styles = StyleSheet.create({
-    topBlock: {
-      ...atoms.py_sm,
-      ...atoms.py_lg,
-    },
+    topBlock: {},
```
Comment (stackchain) [inline apps/wallet-mobile/src/legacy/Dashboard/StakePoolInfo.tsx]: is it needed?
```suggestion
```

**Review by stackchain — APPROVED** [review]

**Pull request merged**